python/lsst/dax/webserv/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'bvan'
```

Comment:
Do we also need `__path__` thing here like in the above files? I guess if we expect other `dax_webserv_*` packages then we do.